package.json

```diff
@@ -45,7 +45,7 @@
     "build": "run-p build:*",
     "build:project": "tsdx build",
     "build:bin": "tsc && rollup -c rollup.config.js",
-    "postbuild": "rimraf dist/**/*.test.*",
+    "prepare": "npm run build",
```

Comment:
Please don't touch this, the deployment to npm is made by github actions 😉 

And this specific line is to avoid having the test part of the npm package (it's an ugly hack, but this is working ^^)

Reply:
I think I deleted it by accident, sorry 😔

Reply:
Happens, this is why we have PR reviews, don't worry 😃

Reply:
@brotheroftux You still need to revert this change 

PS: Sorry for the latency of response, I'm in vacations ^^

Reply:
Removed the `prepare` script, `postbuild` was there. To be fair, I'd say we should let `prepare` stay, since it allows forks to be used as an NPM dependency (https://docs.npmjs.com/files/package.json#git-urls-as-dependencies).